packages/manager/modules/ng-layout-helpers/src/onboarding/translations/Messages_fr_FR.json

```
@@ -0,0 +1,3 @@
+{
+  "pci_project_empty_more": "En savoir plus"
```

```suggestion
  "onboarding_layout_more": "En savoir plus"
```
As we are no longer in pci module

---

This is done. Thanks :)